test/wasm2c/spec/table.txt

```diff
@@ -0,0 +1,5 @@
+;;; TOOL: run-spec-wasm2c
+;;; STDIN_FILE: third_party/testsuite/table.wast
+(;; STDOUT ;;;
+0/0 tests passed.
```

Comment:
Looks like there should be 3 tests in this file.  Does wasm2c not report this number correctly?

Reply:
Those are all `assert_malformed` tests, which wasm2c doesn't run (it's more a test of the binary reader than wasm2c).